apps/fz_http/lib/fz_http_web/user_from_auth.ex

```diff
@@ -20,15 +20,27 @@ defmodule FzHttpWeb.UserFromAuth do
   def find_or_create(%Auth{provider: provider, info: %Auth.Info{email: email}} = _auth)
       when provider in [:google, :okta] do
     case Users.get_by_email(email) do
-      nil -> Users.create_unprivileged_user(%{email: email})
+      nil -> maybe_create_user(auto_create_oidc_users?(), email)
       user -> {:ok, user}
     end
   end
 
   def find_or_create(_provider, %{"email" => email, "sub" => _sub}) do
     case Users.get_by_email(email) do
-      nil -> Users.create_unprivileged_user(%{email: email})
+      nil -> maybe_create_user(auto_create_oidc_users?(), email)
       user -> {:ok, user}
     end
   end
+
+  defp maybe_create_user(true, email) do
```

Comment:
I'd call `auto_create_oidc_users` in here with an if, but not too fussed. Fine either way.